src/components/TableFilterSearch/index.css

```diff
@@ -0,0 +1,50 @@
+.filter-search {
```

Comment:
not sure if we strictly follow this convention, but we often use the same name for css class and component (at first glance we do in v2 components)

can we use `.TableFilterSearch` here?